.github/workflows/jax-vllm-offloading-eks-transfer.yml

```diff
@@ -0,0 +1,52 @@
+name: JAX-vLLM offloading transfer (EKS)
+
+on:
+  workflow_call:
+    inputs:
+      JAX_VLLM_OFFLOADING_IMAGE:
+        type: string
+        description: MaxText image from ghcr.io/nvidia
```

Comment:
This needs update. How about "JAX inference offloading bridge image from ghcr.io"?